src/django/api/management/commands/remote.py

```diff
@@ -0,0 +1,164 @@
+import boto3
+from django.core.management.base import BaseCommand
+
+class Command(BaseCommand):
+    help = 'Run a one-off management command on an ECS cluster.'
```

Comment:
I don't know how I feel about `remote` as the name of this command. It was the best I could come up with in less than 5 minutes; if you have anything better, let me know!

Reply:
`ecsmanage` is another suggestion that incorporates the fact that it is still running a management command and that the command will be running on an ECS instance.

Reply:
That seems much clearer, I'll go with that.